webapi/CopilotChat/Skills/ChatSkills/SemanticChatMemoryExtractor.cs

```diff
@@ -114,35 +114,43 @@ internal static class SemanticChatMemoryExtractor
     /// <param name="context">The context that contains the memory</param>
     /// <param name="memoryName">Name of the memory</param>
     /// <param name="options">The prompts options.</param>
-    internal static async Task CreateMemoryAsync(
-        SemanticChatMemoryItem item,
+    /// <param name="logger"></param>
```

Comment:
Nit: please fill this description:

```suggestion
    /// <param name="logger">Logger</param>
```